.travis.yml

```
@@ -2,14 +2,13 @@ language: ruby
 cache: bundler
 
 addons:
-  postgresql: '9.4'
+  postgresql: '10'
 
 rvm:
-  - ruby-head
-  - 2.5.1
-  - 2.4.4
+  - 2.6.3
```

Should we maybe bump this to `2.6.4`

---

http://www.ruby-lang.org/en/news/2019/10/01/ruby-2-6-5-released/

```suggestion
  - 2.6.5
```